BeatSaberTheater/VideoMenu/VideoMenuUI.cs

```diff
@@ -399,7 +399,10 @@ public void SetupLevelDetailView(VideoConfig videoConfig)
                 break;
             case DownloadState.Converting:
                 _levelDetailMenu.SetActive(true);
-                _levelDetailMenu.SetText($"Converting...",
+                var convertingText = videoConfig.ConvertingProgress.HasValue
```

Comment:
Could you move this into a helper method? That way we only have one place we do this